internal/core/src/pb/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@
 # is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 # or implied. See the License for the specific language governing permissions and limitations under the License
 
+find_package( Protobuf REQUIRED )
```

Comment:
in my environment, it find proto on my laptop thus failed to compile.

Maybe specify find package to use conan first? and print out the proto version

Reply:
remove PROTO_GOOGLE_PATH in this file since it's no longer necessary